lametro/management/commands/compile_pdfs.py

```diff
@@ -221,14 +176,56 @@ def findEventAgendaPacket(self, all_documents=False):
                     ON i.bill_id=d_bill.bill_id
                     INNER JOIN councilmatic_core_eventdocument as d_event
                     ON i.event_id=d_event.event_id
-                    INNER JOIN councilmatic_core_bill AS b
-                    ON d_bill.bill_id=b.ocd_id
-                    WHERE d_bill.bill_id in ( {} )
-                    ) AS subq
+                ) AS subq
```

Comment:
This is a good candidate for a https://www.postgresql.org/docs/9.1/static/queries-with.html 

Let me know if you want some help writing that.